editor.planx.uk/src/@planx/components/Send/uniform/UniformPayload/model.ts

```diff
@@ -0,0 +1,302 @@
+import { XMLBuilder, XmlBuilderOptions } from "fast-xml-parser";
+
+import { Store } from "../../../../../pages/FlowEditor/lib/store/index";
+import { GovUKPayment } from "../../../../../types";
+import { Address } from "../../../AddressInput/model";
+import { GOV_PAY_PASSPORT_KEY } from "../../../Pay/model";
+import { SiteAddress } from "./../../../FindProperty/model";
+import {
+  ApplicantOrAgent,
+  ExistingUseApplication,
+  ExternalAddress,
+  FileAttachment,
+  IUniformPayload,
+  Payment,
+  Proposal,
+  ProposedUseApplication,
+} from "./types";
+
+/**
+ * Available values for passport variable "application.type"
+ */
+export type PlanXAppTypes = "ldc.existing" | "ldc.proposed";
+
+export class UniformPayload implements IUniformPayload {
+  sessionId: string;
+  passport: Store.passport;
+  files: string[];
+
+  proposalCompletionDate: string;
+  siteAddress: SiteAddress;
+
+  "portaloneapp:Proposal": Proposal;
```

Comment:
this is a smart simple way to cheat namespaces! i had definitely gotten very hung up trying to get them typed properly without any success :rabbit: :hole:

Reply:
Yeah I spent a lot of time trying to get to the bottom of this without success and settled on this as a pragmatic step in the right direction. Not one I fancy revisiting in a hurry tbh..!